cfy_manager/components/restservice/restservice.py

```diff
@@ -142,34 +142,20 @@ def _verify_restservice():
     that also requires the storage backend to be up, so if it works, there's
     a good chance everything is configured correctly.
     """
-    rest_port = config[RESTSERVICE]['port']
-    url = 'http://{0}:{1}/api/v2.1/blueprints'.format('127.0.0.1', rest_port)
-
-    wait_for_port(rest_port)
-    req = urllib2.Request(url, headers=get_auth_headers())
-
-    try:
-        response = urllib2.urlopen(req)
-    # keep an erroneous HTTP response to examine its status code, but still
-    # abort on fatal errors like being unable to connect at all
-    except urllib2.HTTPError as e:
-        response = e
-    except urllib2.URLError as e:
-        raise NetworkError(
-            'REST service returned an invalid response: {0}'.format(e))
-    if response.code == 401:
-        raise NetworkError(
-            'Could not connect to the REST service: '
-            '401 unauthorized. Possible access control misconfiguration'
-        )
-    if response.code != 200:
-        raise NetworkError(
-            'REST service returned an unexpected response: '
-            '{0}'.format(response.code)
-        )
+    auth_headers = get_auth_headers()
+    # 'curl' must be run with 'sudo' because the socket is most likely
+    # not readable by the current user.
```

Comment:
`common.sudo()` instead of `run()` perhaps?

Reply:
@tehasdf done.